R/execution.r

```diff
@@ -126,7 +147,12 @@ execute = function(request) {
     err <<- list()
     
     handle_error <- function(e) {
-        err <<- list(ename = 'ERROR', evalue = toString(e), traceback = list(toString(e)))
+        calls <- head(sys.calls()[-seq_len(23)], -3)
```

Comment:
OK, that definitely merits an explanatory comment.

Is there any sane way to calculate the 23 more automatically? I'm worried that if anything changes in our code or evaluate, people could start losing their stack frames.